src/server/protocol.ts

```diff
@@ -585,6 +593,10 @@ namespace ts.server.protocol {
         errorCodes?: number[];
     }
 
+    export interface GetCombinedCodeFixRequestArgs extends FileRequestArgs {
+        groupId: {};

```

Comment:
Why would not this be `string`? i am assuming to allow for an array, but when would you need this? u have an example in mind?